modules/installation-guide/partials/proc_installing-cert-manager.adoc

```diff
@@ -63,8 +63,7 @@ spec:
           # ref: https://docs.microsoft.com/en-us/azure/container-service/kubernetes/container-service-kubernetes-service-principal
           clientSecretSecretRef:
             name: azuredns-config
-            # CLIENT_SECRET is the value of AZURE_CERT_MANAGER_SP_PASSWORD
```

Comment:
Is there any reason the comment above?

Reply:
Hello @tolusha , yes I referenced the Issuer definition file in the original cert-manager documentation and the client secret is using environment variable instead of hard-coded.